components/VcDetails.strings.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
hi @pmigueld whats the difference between these json string and strings that are inside locales i18n? 😄

Reply:
We only edit the `*.strings.json` files. It's compiled using `scripts/compile-strings.js` for localization. There's also a lint-stage rule in package.json to run it when those files are staged.